components/app/job-overview/JobOfferList.vue

```diff
@@ -1,151 +1,60 @@
-            loan: "50000€ - 60000€"
-          },
-          {
-            title: 'poseidon',
-            job: 'Data analyst',
-            description: 'Data analyst which is on track with registring new data assets and maintaining them aswell.',
-            projectTarget: '',
-            imageURL: require('@/assets/images/poseidon.png'),
-            loan: "50000€ - 60000€"
-          },
-          {
-            title: 'poseidon',
-            job: 'Data scientist',
-            description: 'Data analyst who is on track with registring new data assets and maintaining them aswell.',
-            projectTarget: '',
-            imageURL: require('@/assets/images/poseidon.png'),
-            loan: "20000€ - 30000€"
-          },
-          {
-            title: 'poseidon',
-            job: 'Content creator',
-            description: 'Data analyst who is on track with registring new data assets and maintaining them aswell.',
-            projectTarget: '',
-            imageURL: require('@/assets/images/poseidon.png'),
-            loan: "10000€ - 90000€"
-          },
-          {
-            title: 'poseidon',
-            job: 'Rocket Scientist',
-            description: 'Data analyst who is on track with registring new data assets and maintaining them aswell.',
-            projectTarget: '',
-            imageURL: require('@/assets/images/poseidon.png'),
-            loan: "50000€ - 60000€"
-          },
-          {
-            title: 'poseidon',
-            job: 'Data analyst',
-            description: 'Data analyst which is on track with registring new data assets and maintaining them aswell.',
-            projectTarget: '',
-            imageURL: require('@/assets/images/poseidon.png'),
-            loan: "50000€ - 60000€"
-          },
-          {
-            title: 'poseidon',
-            job: 'Data scientist',
-            description: 'Data analyst who is on track with registring new data assets and maintaining them aswell.',
-            projectTarget: '',
-            imageURL: require('@/assets/images/poseidon.png'),
-            loan: "20000€ - 30000€"
-          },
-          {
-            title: 'poseidon',
-            job: 'Content creator',
-            description: 'Data analyst who is on track with registring new data assets and maintaining them aswell.',
-            projectTarget: '',
-            imageURL: require('@/assets/images/poseidon.png'),
-            loan: "10000€ - 90000€"
-          },
-          {
-            title: 'poseidon',
-            job: 'Rocket Scientist',
-            description: 'Data analyst who is on track with registring new data assets and maintaining them aswell.',
-            projectTarget: '',
-            imageURL: require('@/assets/images/poseidon.png'),
-            loan: "50000€ - 60000€"
-          },
-        ],
-      }
-    },
-  }
+  data() {
+    return {
+      jobs: [],
+    }
+  },
+  async fetch() {
+    this.jobs = await getJobs()
```

Comment:
issue (blocking): how do you deal with exceptions? Imagine you cannot call the API. getJobs() does not handle any exceptions atm.